dropwizard-testing/src/test/java/io/dropwizard/testing/junit/TestEntity.java

```diff
@@ -1,13 +1,7 @@
 package io.dropwizard.testing.junit;
 
 import javax.annotation.Nullable;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
-import javax.validation.constraints.NotNull;
+import javax.persistence.*;
```

Comment:
Please don't use wildcard imports but name all imports explicitly.

Reply:
I addressed your comments, have to look at other failing issues.